content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Internal telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+You can check the health of any Collector instance by checking its own internal
+telemetry. Read on to learn how to use this data to diagnose and troubleshoot
+Collector issues.
+
+## Activate internal telemetry in the Collector
+
+By default, the Collector exposes its own telemetry in two ways:
+
+- Internal [metrics](#configure-internal-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configure-internal-logs) are emitted to `stdout` by default.
+
+[Traces](#configure-internal-traces) are not exposed by default. Two feature
+gates offer experimental support for a configuration based on the OpenTelemetry
+Configuration schema.
+
+### Configure internal metrics
+
+Prometheus metrics are exposed locally on port `8888` and path `/metrics`. For
+containerized environments, you might want to expose this port on a public
+interface instead of only locally.
+
+Set the address in the config `service::telemetry::metrics`:
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      address: ':8888'
+```
+
+To visualize these metrics, you can use the
+[Grafana dashboard](https://grafana.com/grafana/dashboards/15983-opentelemetry-collector/),
+for example.
+
+You can enhance the metrics telemetry level using the `level` field. The
```

Comment:
Can we point to a complete list of the metrics for each level?

Reply:
I'm not aware of such a list. Do you have one in mind that you could link here?

Reply:
OK, thanks for clarifying and I don't have such a list, hence I was wondering :)

@svrnm any suggestions whom from SIG Collector to loop in here?

Reply:
tagging @open-telemetry/collector-approvers, please take a look!

If this takes some time to compile that list we can put this into an issue and by that a follow up PR

Reply:
I think we should do this on a separate PR, we are working on that on open-telemetry/opentelemetry-collector/pull/9767

Reply:
@tiffany76 create a new issue referencing this discussion and then you can mark this conversation as resolved

Reply:
Issue created (#4278). Closing conversation.